.circleci/config.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       - setup_remote_docker
       - run: go get -t ./...
       - run: docker swarm init
-      - run: env DAEMON.IMAGE=mesg/daemon:local go test -timeout 30s -p 1 -coverprofile=coverage.txt ./...
+      - run: env DAEMON.IMAGE=mesg/daemon:local go test -timeout 60s -p 1 -coverprofile=coverage.txt ./...
```

Comment:
Because 30s is very close to some package test. I prefer to increase the timeout than having error because of timeout. Again it's depend on the CircleCI remote docker speed.